### DISCUSSION TITLE
Need Help: Failing Parsing of Test Files

### DISCUSSION THREAD

**Discussion by CRasico** [nvim-neotest/neotest#191] [Q&A]
Since upgrading to Neovim 0.8.2 I've noticed some issues in which my instance of neotest is unable to parse the test cases within the individual test files.  In the below example, I'm utilizing neotest and neotest-jest in order to parse all of the jest cases within the repository.
<img width="1270" alt="image" src="https://user-images.githubusercontent.com/53454804/211664239-6bdc4007-6cfc-4f9a-8576-abe86ae94dfa.png">

The issue seems to be related to something in neotest core or possibly the ability to load neovim's internal parser.

Treesitter Installation: 
<img width="988" alt="image" src="https://user-images.githubusercontent.com/53454804/211664631-a2c51e54-0680-4407-a3a4-cb1388750d09.png">

Treesitter Parsers Installed:
<img width="563" alt="image" src="https://user-images.githubusercontent.com/53454804/211664741-47b09f2c-ae23-4b08-91fa-ee229aa0f2a6.png">

Sample Neotest Log: 
[neotest-1.log](https://github.com/nvim-neotest/neotest/files/10386944/neotest-1.log)

Not sure if anyone has seen this issue before, would love any assistance if possible.

**Comment by rcarriga** [comment]
Looks like a similar issue to https://github.com/nvim-neotest/neotest/issues/187#issuecomment-1376499288. I'd suggest a clean re-install as suggested of treesitter, and then otherwise try using the minimal init provided in the bug report to make it easier to diagnose
